.gitignore

```diff
@@ -130,3 +130,5 @@ out
 
 .DS_Store
 dist/
+
+.vscode
```

Comment:
Unrelated to the PR, but I had to stop my vscode from trying to format files with Prettier and removing random trailing whitespace.
Happy to remove that change and keep it for a separate PR!